docs/release/release-team-onboarding.md

```diff
@@ -0,0 +1,62 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
```

Comment:
```suggestion
```

Reply:
Yeah, removed now. It was added automatically when I run doctoc script locally 🤷🏼‍♂️